src/converter/one/jfr/JfrReader.java

```diff
@@ -460,6 +461,7 @@ private void readThreads(int fieldCount) {
             long javaThreadId = getVarlong();
             readFields(fieldCount - 4);
             threads.put(id, javaName != null ? javaName : osName);
+            javaThreads.put(id, javaThreadId);
```

Comment:
Micro-optimization: `threads` dictionary is pre-sized to the required capacity in the beginning of this method, so can `javaThreads`.

Reply:
Both are fine, but I agree Option 2 is a bit easier to understand.

Reply:
Went with option 2. Thanks again for the quick feedback.